cbor.js

```diff
@@ -23,9 +23,63 @@
  */
 
 (function(global, undefined) { "use strict";
-var POW_2_24 = 5.960464477539063e-8,
-    POW_2_32 = 4294967296,
-    POW_2_53 = 9007199254740992;
+var POW_2_32 = 4294967296,
+    POW_2_53 = 9007199254740992,
+    singleFloatView = new Float32Array(1),
+    singleIntView = new Uint32Array(singleFloatView.buffer);
```

Comment:
the constant for POW_2_24 was removed here as it went unused in coverage; it would need to be re-added if encoding and decoding of half-precision floats moves back to using it